src/integrations/garmr.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  bulkhead,
+  circuitBreaker,
+  ConstantBackoff,
+  handleAll,
+  IDefaultPolicyContext,
+  IPolicy,
+  Policy,
+  retry,
+  SamplingBreaker,
+  wrap,
+} from 'cockatiel';
+
+export class GarmrService {
+  instance: IPolicy;
+
+  constructor({
+    handler,
+    breakerOpts,
+    retryOpts,
+    limits,
+  }: {
+    handler?: Policy;
+    breakerOpts: {
+      halfOpenAfter: number;
+      threshold: number;
+      duration: number;
+    };
+    retryOpts?: {
+      maxAttempts: number;
+      timeout: number;
+    };
+    limits?: {
+      maxRequests: number;
+      queuedRequests?: number;
```

Comment:
Max in progress requests that breaker can handle, eg. if set to 100 on the 101th request that happened in parallel it will break.